### DISCUSSION TITLE
How to configure the page title/name on browser tabs?

### DISCUSSION THREAD

**Discussion by Blackmesa-Canteen** [refinedev/refine#4612] [Q&A] [answered]
Hello, could you please tell me that how can to configure the page title/name on browser tabs? 

The problem is that page names in browser's tabs are like "refine",  Invoices | refine" and so on, can I modify it?
<img width="188" alt="image" src="https://github.com/refinedev/refine/assets/69796042/07742b95-4640-44ce-9248-5330e40de234">

Thank you very much!

**Comment by omeraplak** [comment, marked as answer]
Hey @Blackmesa-Canteen,
Thank you for reaching out to us. You can find more details in this document: https://refine.dev/docs/packages/documentation/routers/react-router-v6/#documenttitlehandler
